wire-ios-data-model/Source/Model/Message/ZMClientMessage+Encryption.swift

```diff
@@ -19,7 +19,7 @@
 import Foundation
 import WireCryptobox
 
-private var zmLog = ZMSLog(tag: "message encryption")
+private var zmLog = ZMSLog(tag: "proteus")
```

Comment:
I wouldn't change this, we only want to change the name for the new logs and not the existing ones.